SDK/AppCenter/Microsoft.AppCenter.UWP/Utils/ApplicationStoragePathHelper.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Microsoft.AppCenter.Utils
+{
+    public class ApplicationStoragePathHelper
+    {
+        public static string ApplicationStoragePath => "";
```

Comment:
Why create a storage path helper which just has an empty path constant?

Reply:
It needs to have the same interface as the other platforms since we use shared code